src/widget/wvumetergl.cpp

```diff
@@ -128,8 +126,6 @@ void WVuMeterGL::onConnectedControlChanged(double dParameter, double dValue) {
         // A 0.0 value is very unlikely except when the VU Meter is disabled
         m_dPeakParameter = 0;
     }
-
-    updateState(m_timer.restart());
```

Comment:
In wvumetergl.h, `m-timer` can be removed, and `util/performancetimer.h` be replaced with `util/duration.h`

Reply:
done